### DISCUSSION TITLE
Heritability Analysis

### DISCUSSION THREAD

**Discussion by sandeepgowdan** [gaynorr/AlphaSimR#182] [Q&A]
Understanding heritability (H^2), Variance Explained (VarE), and mean values for different generations (F1, F2, F3, etc.) seems crucial. Could you provide guidance on how to select appropriate values based on previous breeding experiences or studies?

**Comment by gregorgorjanc** [comment]
This is a fundamental aspect of many breeding programmes and you should largely get this information by working with breeders - they have a very good grasp of the variation at different stages of their breeding programmes. It is hard to pull these numbers out of thin air without prior knowledge, though you can work from the basic principles of quantitative genetics and the number of records (=replicates, = plots) that pertain to a genotype.
